README.md

```diff
@@ -53,8 +58,9 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
-
+O(n). The parser would have to store each tile until the first pair of tiles are
```

Comment:
# 3

To be honest, I think it's most complicated than you describe, as there'd need to be backtracking. But the overall idea is right.